code/modules/projectiles/gun_system.dm

```diff
@@ -499,28 +499,33 @@
 	. += overlay
 
 /obj/item/weapon/gun/update_item_state()
-	if(!CHECK_BITFIELD(flags_gun_features, GUN_SHOWS_AMMO_REMAINING))
+	var/current_state = item_state
+	//If the gun has item states that show how much ammo is remaining
+	if(CHECK_BITFIELD(flags_gun_features, GUN_SHOWS_AMMO_REMAINING))
```

Comment:
use & instead of defines pls

Reply:
Updated. When if ever is it appropriate to use CHECK_BITFIELD?